package.json

```diff
@@ -4,38 +4,26 @@
     "@nypl/nypl-core-objects": "^1.2.0",
     "@nypl/nypl-data-api-client": "^0.2.5",
     "@nypl/nypl-streams-client": "^0.1.4",
-    "avro-js": "^1.8.0",
-    "avro-parser": "^1.0.1",
     "avsc": "^4.1.9",
     "aws-sdk": "^2.96.0",
-    "bluebird": "^3.5.0",
+    "bluebird": "^3.7.2",
     "highland": "^2.10.0",
-    "image-size": "^0.5.0",
-    "log4js": "^1.1.0",
     "loglevel": "^1.4.1",
     "mkdirp": "^0.5.1",
-    "mongodb": "^2.2.10",
     "ndjson": "^1.5.0",
-    "oauth": "^0.9.15",
     "optimist": "^0.6.1",
-    "orm": "^3.1.0",
     "pg": "^6.1.0",
-    "pluralize": "^3.1.0",
     "prompt": "^1.0.0",
-    "q-orm": "0.0.3",
-    "request": "^2.75.0",
-    "request-promise": "^4.1.1",
-    "util": "^0.10.3"
```

Comment:
I removed a number of packages that are no longer being used anywhere in the function. Should just make the travis build process faster and reduce the size of the deployed lambda package (which has no practical effect but is still nice I think).